gcsio/src/main/java/com/google/cloud/hadoop/gcsio/storageapioverride/ObjectsGetMedia.java

```diff
@@ -0,0 +1,29 @@
+package com.google.cloud.hadoop.gcsio.storageapioverride;
+
+import com.google.api.services.storage.Storage;
+import com.google.api.services.storage.Storage.Objects.Get;
+import com.google.api.services.storage.model.StorageObject;
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * Overrides Storage.Objects.Get and provides specific functions for getting object media to
+ * decouple getting object media from getting metadata.
+ */
+public class ObjectsGetMedia extends Get {
```

Comment:
I think that `Media` could be a quite specific term for GCS JSON API. May you check GCS documentation on how do they refer to data/metadata requests? If they use `data` then I think it will be more clear to use `Data` instead of `Media` here too.